packages/theming-react-native/src/ThemeProvider/ThemeProvider.tsx

```diff
@@ -22,7 +24,13 @@ export const ThemeProvider: React.FunctionComponent<IThemeProviderProps> = (prop
     };
   }, [registryToUse, themeName, setThemeState]);
 
-  const themeProvider = <ThemeContext.Provider value={theme}>{children}</ThemeContext.Provider>;
+  const boxStyle = getBoxStyle(theme);
```

Comment:
Why not use the hook, and not export `getBoxStyle`?

Reply:
I can't use the hook here.  The ThemeProvider is what's setting the ThemeContext and useTheme here is before the context has been set, so useBoxStyle runs on an undefined theme object.  This is part of the reason I had Box as a separate component initially, because rendering <Box> underneath <ThemeContext.Provider> guaranteed a ThemeContext was initialized by the time the Box was rendered.

Reply:
ah right, I didn't read carefully
I was concerned with exporting both the hook and the non-hook out of `index`, that I didn't notice that the context wasn't set up yet.